src/middlewared/middlewared/plugins/disk.py

```diff
@@ -91,6 +93,9 @@ async def disk_extend(self, disk, context):
         if context['real_names']:
             disk['real_name'] = context['identifier_to_name'].get(disk['identifier'])
 
+        if context['sed_status'] and disk['sed']:
+            disk['sed_status'] = await sed_status(disk['name'])
```

Comment:
Should add this to all disks maybe but have it null or something similar to how passwords above are handled?